### ISSUE TITLE
Range.insertNode does not produce expected result when used on elements with shadow dom

### ISSUE THREAD

**Issue by eeid26** [webcomponents/webcomponentsjs#277]
sample showing the issue
http://jsbin.com/furenogase/2/edit

Please note the insertNode should work with document fragment also.

A PR for this issue is ready to be checked in.

**Comment by addyosmani** [comment]
Fixed by #264?

**Comment by eeid26** [comment]
#264 does not fix this issue (#277).  #264 fixes #268 
I have not created a PR for this issue. Once my current opened PR is merged I will create another PR for #264 and  #276

**Comment by TimvdLippe** [comment]
This is still an issue: http://jsbin.com/kotovok/2/edit?html,console,output

**Comment by TimvdLippe** [comment]
This issue was moved to webcomponents/shadydom#227
